lib/pghero.rb

```diff
@@ -135,6 +136,13 @@ def config
             "databases" => databases
           }
         end
+
+        @isc ||= PgHero::ISC.new
+        isc_pghero_config = @isc.pghero_config
+        Rails.logger.info("[PGBouncerHero] isc_pghero_config is loaded: #{not isc_pghero_config.nil?}")
+        Rails.logger.info("config is: #{isc_pghero_config}")
+        config.merge!(isc_pghero_config) unless isc_pghero_config.nil?
```

Comment:
I think we want to define a method like this
```ruby
module PgHero
    def self.process_config(&block)
        @@dynamic_config = block
    end

    def config
        # keep stuff above
        @@dynamic_config.call(config) if @@dynamic_config.present?
    end
end
```

This way we can define the business logic elsewhere
```ruby
PgHero.process_config do |config|
  # merge with isc config
end
```